### DISCUSSION TITLE
Dependency on encode/databases

### DISCUSSION THREAD

**Discussion by stephane** [codemation/pydbantic#31] [General]
With SQLAlchemy v1.4, I think encode/databases is no longer relevant (the transaction test wrapper still cool BTW).

Do you have reasons to prefer this wrapper against direct SQLAlchemy use? 

Thank you for your feedback.

**Comment by codemation** [comment]
Hey @stephane ,

SQLAlchemy v1.4 was a pretty big step, so decided against removing additional dependencies during this process. The main reason for `databases` is to handle handle the DB connection pooling and tear-down between (MySQL, Postgres, sqlite). Additionally `pydbantic` is meant to be `async` only, a decision that will not likely be reversed soon.
